<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseWidget</class>
 <widget class="QWidget" name="LicenseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>174</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="onlinePage">
      <attribute name="title">
       <string>Internet Activation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="onlineKeyLabel">
         <property name="text">
          <string>License Key</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="onlineKeyEdit">
         <property name="inputMask">
          <string notr="true">&gt;NNNN-NNNN-NNNN-NNNN;_</string>
         </property>
         <property name="text">
          <string notr="true">---</string>
         </property>
         <property name="cursorPosition">
          <number>19</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pasteKeyButton">
         <property name="text">
          <string>Paste from Clipboard</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <item>
          <widget class="QnBusyIndicatorButton" name="activateLicenseButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>License will be bound to the current server</string>
           </property>
           <property name="text">
            <string>Activate License</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
         <property name="topMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QnBusyIndicatorButton" name="activateFreeLicenseButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>License will be bound to the current server</string>
           </property>
           <property name="text">
            <string>Activate Free License</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="licenseKeyWarningLabel">
         <property name="text">
          <string>Invalid License Key</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="manualPage">
      <attribute name="title">
       <string>Manual Activation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="hardwareIdEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="hardwareIdLabel">
         <property name="text">
          <string>Hardware ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="copyHwidButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Copy to Clipboard</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="browseLicenseFileButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="fileLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Select License File...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="activationKeyFileLabel">
         <property name="text">
          <string>Activation Key File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
         <item>
          <widget class="QnBusyIndicatorButton" name="activateLicenseButtonCopy">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>License will be bound to the current server</string>
           </property>
           <property name="text">
            <string>Activate License</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="manualActivationTextWidget">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnBusyIndicatorButton</class>
   <extends>QPushButton</extends>
   <header location="global">ui/widgets/common/busy_indicator_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>serialKeyChanged(QString)</slot>
 </slots>
</ui>
